src/components/Checkbox/README.md

```diff
@@ -134,6 +134,24 @@ function CheckboxExample() {
 
 <!-- /content-for -->
 
+### Checkbox with label hidden
```

Comment:
Added this for verifying in Storybook, but not sure how that will affect the styleguide and if we want it included there.